raspberry_pi/voc_co2/CCS811_logger.py

```diff
@@ -9,13 +9,16 @@ def write2csv(data):
 	log.writerow(data)
 	dataCSV.close()
 
+###### Initialization for the CCS811 sensor from the Adafruit example.py ######
 ccs =  Adafruit_CCS811()
 
 while not ccs.available():
 	pass
 temp = ccs.calculateTemperature()
 ccs.tempOffset = temp - 25.0
 
+
+#adds the file header for new csv
 file_exist = os.path.isfile('CCS811_log.csv')
 if not file_exist:
 	write2csv(['Timestamp', 'CO2 (ppm)', 'TVOC (ppb)', 'Temperature (C)'])
```

Comment:
Let's use database friendly headers. Instead of adding parenthesis to specify the units and upper case, please use lower case on everything and separate units as underline, like so:

timestamp, co2_ppm, tvoc_ppb, temperature_c